template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Views/Account/Login.cshtml

```diff
@@ -59,7 +60,8 @@
             <h4>Use another service to log in.</h4>
             <hr />
             @{
-                var loginProviders = SignInManager.GetExternalAuthenticationSchemes().ToList();
+                var schemes = await SchemeProvider.GetAllSchemesAsync();
```

Comment:
Well we need something, I guess for now, lets just bring back what we had, I'll add DisplayName to AuthenticationScheme and have the Auth's set it like before, and resurrect the SignInManager API for Preview 1, and we can figure out a longer term plan for Preview 2 if need